migrations/Version20230202151933.php

```diff
@@ -10,18 +10,21 @@
 /**
  * Auto-generated Migration: Please modify to your needs!
  */
-final class Version20230123232937 extends AbstractMigration
+final class Version20230202151933 extends AbstractMigration
 {
     public function getDescription() : string
     {
-        return 'Add funder source';
+        return '';
     }
 
     public function up(Schema $schema) : void
     {
         // this up() migration is auto-generated, please modify it to your needs
         $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
 
+        $this->addSql('ALTER TABLE dataset_submission ADD additional_funders TEXT DEFAULT NULL');
+        $this->addSql('ALTER TABLE dataset_submission ALTER cold_storage_total_unpacked_size TYPE INT');
```

Comment:
We're gonna need a bigger boat, see https://www.postgresql.org/docs/14/datatype-numeric.html Since INT is -2147483648 to +2147483647, this would give us a max cold storage filesize of 2GB. (it's signed) We need bigint.

Reply:
This doesn't belong here, and should be removed.